test/distributed/_tensor/test_optimizers.py

```diff
@@ -102,6 +102,15 @@ def test_adam_1d_sharding(self):
                 "amsgrad": True,
                 "foreach": True,
             },
+            {"lr": 0.1, "fused": True},
```

Comment:
sidecar comment: these tests should take advantage of OptimizerInfo constructs instead of redoing the work of figuring out adam configs, but that's outside the scope of this PR cc @wanchaol see example https://github.com/pytorch/pytorch/blob/main/test/test_optim.py#L787.

Reply:
This sounds great! Glad to know this has been added, we should certainly switch to use a more comprehensive OpInfos to improve the optimizer test coverage and avoid redo the work. filed a issue to keep track of this https://github.com/pytorch/pytorch/issues/125620